package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
please remove the package lock json, we have a yarn lock file already.

Reply:
oops. Removed